app/routes/login.js

```diff
@@ -11,8 +12,17 @@ export default Em.Route.extend(UnauthenticatedRouteMixin, {
         if (!hash.access_token) {
             return null;
         }
-        return this.get('session').authenticate('authenticator:jam-osf-jwt', hash.access_token, hash.expires_in).then((data) => {
-            this.transitionTo('experiments');
-        });
+        return this.get('session').authenticate('authenticator:jam-osf-jwt', hash.access_token, hash.expires_in)
+            .then((data) => {
```

Comment:
Single parameter fat arrow functions don't need `()`s around the param.
`.then(data => ...)`